### DISCUSSION TITLE
[BP#48] Suppression de "Utiliser les opérations primitives"

### DISCUSSION THREAD

**Discussion by devcrafting** [cnumr/best-practices#67] [Bonnes pratiques] [answered]
Cette BP me semble faire partie des micro-optimisations sans grand impact mais avec un effet catastrophique sur la lisibilité du code et sa maintenabilité.

**Comment by didierfred** [comment]
je suis d'accord , ci joint une page test [test.zip](https://github.com/cnumr/best-practices/files/7404723/test.zip)  (Il y a aussi des tests d'autres pratiques)


Je n'ai aucune différence  sur mon PC à l'execution de 50 Millions d'opérations sans ou avec primitive , 4ms chacune  !

**Comment by ACTLEM** [comment, marked as answer]
https://github.com/cnumr/best-practices/issues/125
